src/remote/impl/identity.service.ts

```diff
@@ -78,7 +78,7 @@ export class IdentityService implements IIdentityService {
     public async qualityCheck(id: string): Promise<any> {
         const request: AxiosRequestConfig = {
             method: 'GET',
-            url: this.baseUrl + '/api/v1/positions/' + this.backend + `/did=${id}`,
+            url: this.baseUrl + '/api/v1/positions/' + this.backend + `/dids=${id}`,
```

Comment:
Is this an API change that means we need to deploy different new versions of different services in lockstep or is it changing to use a different API endpoint entirely? Or is this calling the key guardian service itself?

Reply:
This is using an API change in Identity Service that was introduced a little while back and has, I believe, already been deployed (that was the whole needing `MAX_DIDS` environment variable thing): https://github.com/kiva/guardian-bio-auth/pull/12

The change from `did` to `dids` is a fix of an error that I just merged recently and hasn't been deployed yet (thankfully, because it's an error).